### ISSUE TITLE
[Feature]: LN manual channel with external node

### ISSUE THREAD

**Issue by pwltr** [synonymdev/bitkit-react-native#1972]
### Describe the problem

Blocked by https://github.com/synonymdev/react-native-ldk/issues/30

While Bitkit does support LNURL-channel, there is no way to connect to nodes that do not generate such a QR.

### Describe the solution

Add the flow for adding non Blocktank nodes. Also need to make sure open/close transactions are marked as such in the activity.

![image](https://github.com/synonymdev/bitkit/assets/8538369/e2d07229-4eb3-4e6f-aa8a-f6256a88534b)

**Comment by JeanlChristophe** [comment]
jay will do  https://github.com/synonymdev/react-native-ldk/issues/30. it will be available for August. only one day of work.

**Comment by JeanlChristophe** [comment]
https://github.com/synonymdev/react-native-ldk/pull/268

**Comment by catch-21** [comment]
Tested on 1.0.5 (136)

1.
We should allow those in US/Canada to use this feature: https://github.com/synonymdev/bitkit/issues/2252

2.
Crash on iPhone if 'Continue' is pressed without entering any node details in on Manual Setup screen. This then may have broken my LDK node, or coincidentally this occurred. https://github.com/synonymdev/bitkit/issues/2253

3.
Error shown without message on android in same scenario as above (2). Not raised a new issue but added observation here https://github.com/synonymdev/bitkit/issues/2253#issuecomment-2381871931. A fix must be tested on both platforms.
<img src="https://github.com/user-attachments/assets/bc00dab0-7a15-42c6-a88b-8296e35668f6" width="300">

4.
Host and Port are not required to proceed to attempt swipe and transfer, only the Node ID is. Form validation could be improved here for mandatory fields https://github.com/synonymdev/bitkit/issues/2257

5.
Unable to establish a connection with Regtest Blocktank https://github.com/synonymdev/bitkit/issues/2259

-----

- Can paste URI into form
- Can swipe to transfer using mainnet Blocktank details https://1ml.com/node/0296b2db342fcf87ea94d981757fdf4d3e545bd5cef4919f58b5d38dfdd73bf5c9
  - Purple Transfer suggestion card is shown immediately
  - Orange Transfer Activity is shown immediately
  - Spending page indicates incoming transfer for correct amount (no lsp fee, and ignoring network fee)
  - No spending Activity shown
